apps/i18n/common/en_us.json

```diff
@@ -243,7 +243,7 @@
   "completed": "Completed",
   "completedLevels": "Completed Levels",
   "completedLessons": "Completed Lessons",
-  "completedStudentCount": "{numStudentsCompleted} of {numStudents} students completed",
+  "completedStudentCount": "{numStudentsCompleted} of {numStudents} students completed ({percentComplete} %)",
```

Comment:
note that any existing translation of this string won't have the `percentComplete` variable in it until the string gets retranslated

Reply:
How often do stings like this get re-translated? This might be ok since the feature isn't available to the public yet.

Reply:
It depends on translator activity. When a source string gets updated like this, any existing translations will be flagged for retranslation, but whether or not that actually gets done is up to the translators.

For a feature that's not yet available, this is probably fine.